{"error": null, "issues": [{"address": 5233, "code": ";\n\nlibrary SafeMat", "debug": "storage_keccac_calldata_TEFoodsToken_32_+_36: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\ncalldata_TEFoodsToken_32 + 4: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\nstorage_10826069350932453726649810804370223703603575740452467603846993781924376990700: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\ncalldata_TEFoodsToken_32 + 36: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\ncalldata_TEFoodsToken_4: 0xffffffffffffffffffffffffffffffffffffffff\nstorage_0: 0x0\ncaller: 0x0\ncalldata_TEFoodsToken_0: 0xa29cc6c800000000000000000000000000000000000000000000000000000000\ncalldatasize_TEFoodsToken: 0x4\ncallvalue: 0x0\n", "description": "A possible integer overflow exists in the function `_function_0xa29cc6c8`.\nThe addition or multiplication may result in a value higher than the maximum representable integer.", "filename": "/unique_chucks/8/0x940d73c91db9f82440702f6cc8323a8c60583777.sol", "function": "_function_0xa29cc6c8", "lineno": 1, "title": "Integer Overflow ", "type": "Warning"}, {"address": 6780, "code": "  return a - b", "debug": "The exception is triggered under the following conditions:\n\ncalldata_TEFoodsToken_32 + 4: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\nstorage_10826069350932453726649810804370223703603575740452467603846993781924376990700: 0xfffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe\ncalldata_TEFoodsToken_32 + 36: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\ncalldata_TEFoodsToken_4: 0xffffffffffffffffffffffffffffffffffffffff\nstorage_0: 0x0\ncaller: 0x0\ncalldata_TEFoodsToken_0: 0xa29cc6c800000000000000000000000000000000000000000000000000000000\ncalldatasize_TEFoodsToken: 0x4\ncallvalue: 0x0\n", "description": "A reachable exception (opcode 0xfe) has been detected. This can be caused by type errors, division by zero, out-of-bounds array access, or assert violations. This is acceptable in most situations. Note however that `assert()` should only be used to check invariants. Use `require()` for regular input checking. ", "filename": "/unique_chucks/8/0x940d73c91db9f82440702f6cc8323a8c60583777.sol", "function": "_function_0xa29cc6c8", "lineno": 20, "title": "Exception state", "type": "Informational"}, {"address": 6798, "code": "a);\n ", "debug": "timestamp: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffef00\nstorage_0: 0x0\ncaller: 0x0\ncalldata_TEFoodsToken_0: 0xaf35c6c700000000000000000000000000000000000000000000000000000000\ncalldatasize_TEFoodsToken: 0x4\ncallvalue: 0x0\n", "description": "A possible integer overflow exists in the function `_function_0xaf35c6c7`.\nThe addition or multiplication may result in a value higher than the maximum representable integer.", "filename": "/unique_chucks/8/0x940d73c91db9f82440702f6cc8323a8c60583777.sol", "function": "_function_0xaf35c6c7", "lineno": 25, "title": "Integer Overflow ", "type": "Warning"}, {"address": 6811, "code": "rn c;\n  }\n}\n\nc", "debug": "The exception is triggered under the following conditions:\n\ntimestamp: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffb200\nstorage_0: 0x0\ncaller: 0x0\ncalldata_TEFoodsToken_0: 0xaf35c6c700000000000000000000000000000000000000000000000000000000\ncalldatasize_TEFoodsToken: 0x4\ncallvalue: 0x0\n", "description": "A reachable exception (opcode 0xfe) has been detected. This can be caused by type errors, division by zero, out-of-bounds array access, or assert violations. This is acceptable in most situations. Note however that `assert()` should only be used to check invariants. Use `require()` for regular input checking. ", "filename": "/unique_chucks/8/0x940d73c91db9f82440702f6cc8323a8c60583777.sol", "function": "_function_0xaf35c6c7", "lineno": 26, "title": "Exception state", "type": "Informational"}], "success": true}